docs/ref/checks.txt

```diff
@@ -485,6 +485,8 @@ The following checks are run if you use the :option:`check --deploy` option:
   should consider enabling this header to protect user privacy.
 * **security.E023**: You have set the :setting:`SECURE_REFERRER_POLICY` setting
   to an invalid value.
+* **security.W24**: You have set the :setting:`CACHES` ``LOCATION`` setting
+  to directory otherwise used by the application.
```

Comment:
I think we should move this to `caches` i.e. `caches.W002`.

Reply:
This should document a warning message (_"Your cache configuration might expose or corrupt your file system."_), but it's not the same.